src/Codeception/Module/Db.php

```diff
@@ -58,6 +61,9 @@
  *              populate: true
  *              cleanup: true
  *              reconnect: true
+ *              ssl_key: '/path/to/client-key.pem'
+ *              ssl_cert: '/path/to/client-cert.pem'
+ *              ssl_ca: '/path/to/ca-cert.pem'
```

Comment:
You are using absolute paths, right?
I wonder what should we do with relative paths.
@DavertMik should the relative paths be relative to project dir or data dir?

Reply:
Now, relative paths from the project directory are working. Absolute paths should also be supported in the future.

Reply:
Actually they are relative to current working directory, which isn't always the same as project dir (running Codeception with `-c path` and included test suites are two cases when they don't match). But this can be changed later if necessary.